frontend/src/components/common/header/header.tsx

```diff
@@ -1,34 +1,47 @@
 import defaultUserAvatar from 'assets/img/avatar-default.svg';
 import logo from 'assets/img/logo.svg';
+import { AppRoute } from 'common/enums/enums';
 import { FC } from 'common/types/types';
-import { Image } from 'components/common/common';
-import { useState } from 'hooks/hooks';
+import { Button, Image } from 'components/common/common';
+import { useAppSelector, useNavigate, useState } from 'hooks/hooks';
 
 import { Popup } from './components/components';
 import styles from './styles.module.scss';
 
 const Header: FC = () => {
   const [isMenuPopupVisible, setIsMenuPopupVisible] = useState<boolean>(false);
+  const { user } = useAppSelector((state) => state.auth);
+  const navigate = useNavigate();
 
   const handlePopupOper = (): void =>
     setIsMenuPopupVisible(!isMenuPopupVisible);
 
+  const handleLogIn = (): void => navigate(AppRoute.SIGN_IN);
+  const handleSignUp = (): void => navigate(AppRoute.SIGN_UP);
```

Comment:
Maybe `handleNavigateToLogIn` and `handleNavigateToSignUp`? Because the names `handleLogin` and `handleSignUp` don't tell, that you're redirecting, but it's like you're logging the user in or signing them up.